src/components/TableHead.js

```diff
@@ -50,12 +50,12 @@ class TableHead extends React.Component {
 
     const numSelected = (selectedRows && selectedRows.data.length) || 0;
     let isIndeterminate = numSelected > 0 && numSelected < count;
-    let isChecked = numSelected > 0 && numSelected === count;
+    let isChecked = numSelected > 0 && numSelected >= count;
 
     // When the disableToolbarSelect option is true, there can be
     // selected items that aren't visible, so we need to be more
     // precise when determining if the head checkbox should be checked.
-    if (options.disableToolbarSelect === true) {
+    if (options.disableToolbarSelect === true || (options.selectToolbarPlacement === 'none' || options.selectToolbarPlacement === 'above')) {
```

Comment:
```js
(options.disableToolbarSelect === true || **(**options.selectToolbarPlacement === 'none' || options.selectToolbarPlacement === 'above'**)**)
```
Why did you use parentheses?
It's the same result, if or not. because **||**

Reply:
ah, good point! I think I originally grouped them because I thought it would be an && operation, but it's not. Basically if any of those values are true (the regular toolbar being visible when items are selected), then the special case should go forward.

Reply:
@patorjk  good~!